src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionMatchXpathTest.java

```diff
@@ -87,4 +89,322 @@ public void testTwo() throws Exception {
         runVerifications(moduleConfig, fileToProcess, expectedViolation,
                 expectedXpathQueries);
     }
+
+    @Test
+    public void testEncodedOne() throws Exception {
+        final File fileToProcess =
+                new File(getPath("SuppressionXpathRegressionMatchXpathEncodedQuoteString.java"));
+
+        final DefaultConfiguration moduleConfig =
+                createModuleConfig(OperatorWrapCheck.class);
+
+        final String[] expectedViolation = {
+            "5:38: " + getCheckMessage(OperatorWrapCheck.class, OperatorWrapCheck.MSG_LINE_NEW,
+                    '+'),
+        };
+
+        final List<String> expectedXpathQueries = Arrays.asList(
+            "/CLASS_DEF[./IDENT[@text='SuppressionXpathRegressionMatchXpathEncodedQuoteString']]/"
+                    + "OBJBLOCK/VARIABLE_DEF[./IDENT[@text='quoteChar']]/ASSIGN/EXPR",
+            "/CLASS_DEF[./IDENT[@text='SuppressionXpathRegressionMatchXpathEncodedQuoteString']]/"
+                    + "OBJBLOCK/VARIABLE_DEF[./IDENT[@text='quoteChar']]/ASSIGN/EXPR/PLUS"
+                    + "[./STRING_LITERAL[@text='\\&quot;testOne\\&quot;']]"
+        );
+
+        runVerifications(moduleConfig, fileToProcess, expectedViolation,
+                expectedXpathQueries);
+    }
+
+    @Test
+    public void testEncodedTwo() throws Exception {
```

Comment:
Fix names of method, same as you did with class name

Reply:
Done